src/data/polynomial/eval.lean

```diff
@@ -447,6 +447,28 @@ lemma map_prod {ι : Type*} (g : ι → polynomial R) (s : finset ι) :
   (∏ i in s, g i).map f = ∏ i in s, (g i).map f :=
 eq.symm $ prod_hom _ _
 
+lemma map_sum {ι : Type*} (g : ι → polynomial R) (s : finset ι) :
+  (∑ i in s, g i).map f = ∑ i in s, (g i).map f :=
+eq.symm $ sum_hom _ _
+
+lemma map_support_is_subset (p : polynomial R) : (map f p).support ⊆ p.support :=
```

Comment:
```suggestion
lemma support_map_subset (p : polynomial R) : (map f p).support ⊆ p.support :=
```
I think this name is more appropriate: we're applying `support` to `map`, it's only read right-to-left because of the dot notation. Is there precedent for using `is_subset` instead of just `subset`?

Reply:
@Vierkantor I won't stop bors, if you think the name is fine then let it go.

Reply:
I agree that `support_map_support` is more in line with our conventions. This PR is still in the queue, so I'll commit the change and re-approve.